packages/webapp/src/components/Animals/AddAnimalsFormCard/AnimalSelect.tsx

```diff
@@ -47,7 +48,11 @@ export function AnimalTypeSelect<T extends FieldValues>({
       <Controller
         name={name}
         control={control}
-        rules={{ required: { value: true, message: t('common:REQUIRED') } }}
+        rules={{
+          required: { value: true, message: t('common:REQUIRED') },
+          // @ts-ignore
+          validate: hookFormSelectUniquePropertyValidation(typeOptions, 'label', 'Already exists'),
```

Comment:
Shouldn't `'Already exists'` be a translation?

Reply:
Yes haha it should thanks!

Reply:
Updated